packages/mask/src/extension/popups/pages/Friends/Detail/Account/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { memo } from 'react'
```

Comment:
This file looks duplicated with `packages/mask/src/extension/popups/pages/Friends/ContactCard/Account/index.tsx`.